spec/tracebacks/ellipsis/Makefile

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024, The Pallene Developers
+# Pallene Tracer is licensed under the MIT license.
+# Please refer to the LICENSE and AUTHORS files for details
+# SPDX-License-Identifier: MIT 
+
+CC := gcc
+
+.PHONY : all
+.SILENT:
+
+all: 
+	$(CC) -std=c99 -pedantic -Wall -Wextra -fPIC -DPT_DEBUG -shared module.c -o module.so
```

Comment:
These makefiles also will need some love. There should be a "module.so" target that depends on "module.c".
No need for all, since there's a single target.